src/ripple/shamap/SHAMap.h

```diff
@@ -138,6 +138,7 @@ class SHAMap
     bool hasItem (uint256 const& id) const;
     bool delItem (uint256 const& id);
     bool addItem (SHAMapItem const& i, bool isTransaction, bool hasMeta);
+    bool addItem (SHAMapItem&& i, bool isTransaction, bool hasMeta);
```

Comment:
Since `SHAMapItem` has one member that is fast to move from (`data_`) and one that isn't (`tag_`) it is appropriate to have the two overloads (as opposed to my usual suggestion to move from a value).